prismic-slices/index.js

```diff
@@ -1,22 +1,19 @@
 // This file is generated by Slice Machine. Update with care!
 
 import ChartRow from './ChartRow';
-import LearnAndEarnCategory from './LearnAndEarnCategory';
```

Comment:
Should we remove those? Won't it affect the learn and earn feature?

Reply:
Good point, i have no idea..
I basically edited a slice and he deleted that line i don't know why. Should I open a PR with only the slice change?

Reply:
No worries, if needed i'll re-add it, along with the learn and earn logic.